com.unity.render-pipelines.universal/Runtime/2D/LightCachedMeshData.cs

```diff
@@ -0,0 +1,69 @@
+using System.Collections.Generic;
+using System.Linq;
+using Unity.Collections;
+using Unity.Mathematics;
+using UnityEngine.Experimental.Rendering.Universal.LibTessDotNet;
+using UnityEngine.Rendering;
+using UnityEngine.U2D;
+
+namespace UnityEngine.Experimental.Rendering.Universal
+{
+
+    // Simple Cache for Shape Geometry Data.
+    // Note: The cache only gets saved in Editor mode.
+    // On Runtime, data is only used on Load. Any further changes to Light2D will involve regeneration.
+    [AddComponentMenu("")]
+    internal class LightCachedMeshData : MonoBehaviour
```

Comment:
Why does this have to be a standalone MonoBehaviour rather part of the Light2D component itself?

Reply:
The idea is provide this as an optional behavior instead of adding overhead in Light2D. Besides this creates a clear separation of functionality.